todo_app/data/trello_items.py

```diff
@@ -0,0 +1,146 @@
+import os
+from todo_app.data.item import Item, ItemStatus
+from typing import List, Optional
+
+import requests
+
+query_params = {
+    'token': os.environ['TRELLO_API_TOKEN'],
+    'key': os.environ['TRELLO_API_KEY']
+}
+
+base_uri = "https://api.trello.com/1"
+
+def get_todo_board_id() -> str:
+    configured_todo_board_name = os.environ['TRELLO_API_TODO_BOARD_NAME']
```

Comment:
This is slightly more user friendly, but note that we could reasonably save this call every time by just asking the user to provide a board id instead of a board name as an environment variable (and that would arguably be even more resilient, since the name won't necessarily be unique)